idd/Energy+.idd.in

```diff
@@ -21362,10 +21362,10 @@ People,
   A7 , \field Mean Radiant Temperature Calculation Type
        \note optional (only required for thermal comfort runs)
        \type choice
-       \key ZoneAveraged
+       \key EnclosureAveraged
```

Comment:
key change for People. This now uses the enclosure MRT for the associated space. No longer using zone MRT.